e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -32,9 +32,19 @@ const path = require('path');
 const NOTEBOOK_NAME = 'Notebook';
 
 test.describe('Notebook CRUD Operations', () => {
-  test.fixme('Can create a Notebook Object', async ({ page }) => {
+  test.beforeEach(async ({ page }) => {
+    //Navigate to baseURL
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+  });
+  test('Can create a Notebook Object', async ({ page }) => {
     //Create domain object
+    await createDomainObjectWithDefaults(page, {
+      type: NOTEBOOK_NAME
+    });
     //Newly created notebook should have one Section and one page, 'Unnamed Section'/'Unnamed Page'
+    const notebookSectionNames = page.locator('.c-notebook__sections .c-list__item__name');
+    await expect(notebookSectionNames).toBeHidden();
```

Comment:
We should also assert that the sidebar page name element is hidden, and that their text matches what we would expect:

```suggestion
    const notebookSectionNames = page.locator('.c-notebook__sections .c-list__item__name');
    const notebookPageNames = page.locator('.c-notebook__pages .c-list__item__name');
    await expect(notebookSectionNames).toBeHidden();
    await expect(notebookPageNames).toBeHidden();
    await expect(notebookSectionNames).toHaveText('Unnamed Section');
    await expect(notebookPageNames).toHaveText('Unnamed Page');
```